website/server/extractDocs.js

```diff
@@ -283,6 +399,11 @@ const components = [
   '../Libraries/Components/ViewPager/ViewPagerAndroid.android.js',
   '../Libraries/Components/WebView/WebView.ios.js',
 ];
+/*
+const apis = [
+  '../Libraries/ActionSheetIOS/ActionSheetIOS.js',
```

Comment:
Was `ActionSheetIOS` one already in jsdoc format? I don't see it in this pull request.